src/redux/initialState.ts

```diff
@@ -72,6 +72,7 @@ const initialAppState: AppState = {
     searchHistory: electronStore.get('appConfig.recentSearch') || [],
     currentMatch: null,
   },
+  webContentsId: 1,
```

Comment:
maybe this should be `undefined` by default? not sure how well it will work with multiple windows if we set it to `1`